homeassistant/components/light/strings.json

```diff
@@ -1,5 +1,41 @@
 {
   "title": "Light",
+  "common": {
```

Comment:
This could isntead be called "shared", "references", "common_references" etc.

Reply:
As discussed in a meeting, we concluded we use `common` on a few places already; thus the most logical one.